syntax = "proto3";
package Protocol;

import "Enum.proto";

message PosInfo
{
	uint32 object_id = 1;
	double x = 2;
	double y = 3;
	double z = 4;
	double yaw = 5;
	MoveState move_state = 6;
}

message ObjectInfo
{
	uint32 object_id = 1;
	ObjectType object_type = 2;
	PosInfo pos_info = 3;
}